src/main/java/com/google/api/codegen/transformer/java/JavaFeatureConfig.java

```diff
@@ -68,17 +69,27 @@ public static Builder newBuilder() {
   }
 
   @AutoValue.Builder
-  public abstract static class Builder {
+  abstract static class Builder {
```

Comment:
Why are these becoming package private? Just because they can be?

Reply:
Yeah, figured I would adhere to the principle where we restrict visibility if possible; we can always make it public if need be.